### PR TITLE
feat: Added migration for new Jupyterlab Workspace, 1.0.1

### PR DESCRIPTION
**Pull request by aleksandrmelnikov**
**What this PR does**:
Adds migration for new Jupyterlab Workspace.

**Which issue(s) this PR fixes**:

Fixes onepanelio/core#415

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by rushtehrani** [comment]
This should work with the following:

TensorFlow notebook: https://raw.githubusercontent.com/tensorflow/tensorboard/master/docs/graphs.ipynb
PyTorch example: https://pytorch.org/tutorials/intermediate/tensorboard_tutorial.html

**Comment by rushtehrani** [comment]
@aleksandrmelnikov this has a minor conflict.

**Comment by aleksandrmelnikov** [comment]
> @aleksandrmelnikov this has a minor conflict.

Fixed conflict.

**Comment by rushtehrani** [comment]
@aleksandrmelnikov I'll merge once you update to the latest JupyterLab image

File: db/go/20200821162630_update_jupyter_workspace_template.go [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+	v1 "github.com/onepanelio/core/pkg"
+	uid2 "github.com/onepanelio/core/pkg/util/uid"
+	"github.com/pressly/goose"
+)
+
+const jupyterWorkspaceTemplate2 = `# Docker containers that are part of the Workspace
+containers:
+- name: jupyterlab-tensorflow
+  image: onepanel/jupyterlab:1.0.1
+  command: ["/bin/bash", "-c", "mkdir /logs; tensorboard --host 0.0.0.0 --logdir logs & start.sh jupyter lab --LabApp.token='' --LabApp.allow_remote_access=True --LabApp.allow_origin=\"*\" --LabApp.disable_check_xsrf=True --LabApp.trust_xheaders=True --LabApp.base_url=/ --LabApp.tornado_settings='{\"headers\":{\"Content-Security-Policy\":\"frame-ancestors * \'self\'\"}}' --notebook-dir='/data' --allow-root"]
```
Comment (rushtehrani) [inline db/go/20200821162630_update_jupyter_workspace_template.go:14]: @aleksandrmelnikov do we actually need to have tensorboard running? They can run it via the tb magic command.

**Pull request merged**